FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt

```diff
@@ -223,12 +253,18 @@ object PgpMsg {
 
   fun processMimeMessage(
     msg: MimeMessage,
+    pgpPublicKeyRingCollection: PGPPublicKeyRingCollection = PGPPublicKeyRingCollection(listOf()),
```

Comment:
could be shorter and simpler names, just like "publicKeys" and "secretKeys"

Reply:
good to mention purpose in name: verificationPublicKeys